process/smartContract/process.go

```diff
@@ -1657,17 +1720,25 @@ func (sc *scProcessor) useLastTransferAsAsyncCallBackWhenNeeded(
 	return true
 }
 
-func (sc *scProcessor) isTransferWithNoDataOrBuiltInCall(data []byte) bool {
+func (sc *scProcessor) isTransferWithNoAdditionalData(data []byte) bool {
 	if len(data) == 0 {
 		return true
 	}
-	function, _, err := sc.argsParser.ParseCallData(string(data))
+	function, args, err := sc.argsParser.ParseCallData(string(data))
 	if err != nil {
 		return false
 	}
 
 	_, err = sc.builtInFunctions.Get(function)
-	return err == nil
+	if err != nil {
+		return false
+	}
+
+	if function != core.BuiltInFunctionESDTTransfer {
```

Comment:
Is this a case of backwards compatibility?

Reply:
yes it is. this code was not enabled yet.